### PR TITLE
task/CSI-4566 finish PR 256

### PR DESCRIPTION
**Pull request by matancarmeli7**
Signed-off-by: matancarmeli7 <matan.carmeli7@gmail.com>

### CODE REVIEW

File: api/v1/common.go [hunk 1]
```diff
@@ -16,11 +16,11 @@
 
 package v1
 
-type DriverPhase string
+type ProductPhase string
```
Comment (ArbelNathan) [inline api/v1/common.go:19]: is it for [this](https://github.com/IBM/ibm-block-csi-operator/pull/256#discussion_r896975155)?

Comment (matancarmeli7) [reply api/v1/common.go:19]: yes

Comment (ArbelNathan) [reply api/v1/common.go:19]: I think we can stay with `Driver`

Comment (matancarmeli7) [reply api/v1/common.go]: done

**Review by ELENAGER — APPROVED** [review]
LGTM

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**